thunder/dynamo/splitter.py

```diff
@@ -183,9 +207,30 @@ def is_thunder_supported_partition(node: torch.fx.Node) -> bool:
                 partial(_get_example_inputs_from_placeholder, only_metadata=True), placeholders
             )
             example_input_metadatas.append(list(example_input_metadata))
+
+            trace_structured_artifact(
```

Comment:
It would be nice to have a helper called `log_fx_graph` for readability and usage.